### PR TITLE
Suppress error from monaco-editor

### LINKED ISSUE #29414: Uncaught TypeError: e.getModifierState is not a function
### Description

image
I think this is not caused by browser extensions, as I have closed all of them that I could.

Reproduce:
1. Open a repo, and click add new file.
2. At the input box of filename, select `LICENSE` from the list:
image

### Gitea Version

latest

### Can you reproduce the bug on the Gitea demo site?

Yes

### Log Gist

_No response_

### Screenshots

_No response_

### Git Version

_No response_

### Operating System

_No response_

### How are you running Gitea?

build

### Database

None

### PR DESCRIPTION
**Pull request by silverwind**
Fixes: https://github.com/go-gitea/gitea/issues/29414

I see no way for us to catch this error, so downgrade it until https://github.com/microsoft/monaco-editor/issues/4325 is fixed, which will likely take a few weeks to propagate up from vscode.

The entries in `updates.config.js` will make [`updates`](https://github.com/silverwind/updates) not upgrade these anymore and I think it's good documentation as well to have the reasons why we don't upgrade these dependencies.

### CODE REVIEW

File: updates.config.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+export default {
+  exclude: [
+    'monaco-editor', // https://github.com/microsoft/monaco-editor/issues/4325
+    '@mcaptcha/vanilla-glue', // breaking changes in rc versions
+    'eslint-plugin-array-func', // need to migrate to eslint flat config first
```
Comment (silverwind) [inline updates.config.js]:
```suggestion
    '@mcaptcha/vanilla-glue', // breaking changes in rc versions
    'eslint-plugin-array-func', // need to migrate to eslint flat config first
    'monaco-editor', // https://github.com/microsoft/monaco-editor/issues/4325
```

**Review by denyskon — APPROVED** [review]

**Review by lunny — APPROVED** [review]

**Comment by silverwind** [comment]
I fully understand the issue now and I think I will just add a error filter to not display this error as there are no bad outcomes from it.

**Comment by silverwind** [comment]
Rewrote the PR, now it's a simple error filter.

**Comment by silverwind** [comment]
Leaving this open for a bit so people can re-review.

**Review by denyskon — APPROVED** [review]

**Pull request merged**